interface/interface.dm

```diff
@@ -57,6 +57,10 @@
 	set name = "report-issue"
 	set desc = "Report an issue"
 	set hidden = 1
+
+	if(alert(src, "What kind of issue are you facing?", "Report Issue", "In-game Bug", "UI issue", "Other") != "UI issue")
```

Comment:
Categories should be Bug, TGUI Non-Functional, and Cancel

Reply:
I'd rather say 'UI issue' since some players might not know what TGUI is. Most of the text in this PR has been designed for people that don't know anything regarding coding.

Reply:
Cancel button would be nice